App.js

```diff
@@ -24,6 +25,8 @@ LogBox.ignoreLogs(["AsyncStorage has been extracted from"]);
 // Create the navigator
 const Stack = createNativeStackNavigator();
 
+const storage = getStorage(app);
```

Comment:
- This is the line causing the compiling error you're getting. A syntax error.
- The line is unnecessary since you repeat the same instruction on the try-catch block below. Kindly delete it

Reply:
`app` is not defined in the outer scope(outside of the component class).